src/app/models/salesforce.js

```diff
@@ -33,7 +33,8 @@ function adoption(options) {
 const salesforce = {
     adoption,
     adoptionName,
-    salesforceHome: settings.testingEnvironment ? 'test.salesforce.com' : 'webto.salesforce.com'
+    salesforceHome: settings.testingEnvironment ? 'test.salesforce.com' : 'webto.salesforce.com',
+    oid: settings.testingEnvironment ? '00Dc0000003vv3Q' : '00DU0000000Kwch'
```

Comment:
Could/should the settings file contain the salesforce id directly? This way, the settings.js file contains the specific keys should they need to change in the future. Similar to the webview settings.js file . Also:

1. if different environments needed to test a different salesforce id's they could without having to change this code
2. the testing salesforce id is not downloaded by every visitor to the website

Reply:
If we put it in settings.js, it becomes a devops change rather than a code change, and it has to be changed in each of the deployment environments.